external-crates/move/crates/move-prover/tests/integration.rs

```diff
@@ -89,7 +89,15 @@ fn run_prover(file_path: &PathBuf) -> String {
     // Now handle the result of our operation
     match result {
         Ok(output) => output,
-        Err(_) => "Verification failed: panic during verification".to_string(),
```

Comment:
now that the code doesn't `panic` to report errors, are any of the changes in this file needed? If not, let's undo them.